pkg/virt-controller/watch/migration.go

```diff
@@ -746,6 +746,7 @@ func (c *MigrationController) handleTargetPodCreation(key string, migration *vir
 
 	// XXX: Make this configurable, think about limit per node, bandwidth per migration, and so on.
 	if len(runningMigrations) >= int(*c.clusterConfig.GetMigrationConfiguration().ParallelMigrationsPerCluster) {
+		log.Log.Object(migration).Infof("Waiting to schedule target pod for vmi [%s/%s] migration because total running parallel migration count [%d] is currently at the global cluster limit.", vmi.Namespace, vmi.Name, len(runningMigrations))
```

Comment:
How do you feel about adding events too? Of course, it would not be ideal to trigger it every 5 seconds.

Consider a case where a user can't know how many migrations are in flight.

Reply:
I would say that the user should have very little interest in migrations at all. The workload is not interrupted during a migration. Therefore, the alert should be redundant.

> Consider a case where a user can't know how many migrations are in flight.

Migration is an administrators' task. The admin should be able to know.

Reply:
How about associating the events with the migration objects, then the admins get the info and not the users, since the migration object is for admins.

Reply:
> How do you feel about adding events too? Of course, it would not be ideal to trigger it every 5 seconds.

I think the default inhibition in the event client and the merging of events on the cluster-level may be sufficient.

Reply:
Fair enough my example case was not best :) The user is the admin in this case but I admit that the admin should have access to all migrations and therefore see how many are in flight. Therefore the event acts as additional information for individual migration objects.

Reply:
What I don't understand is what action do we expect the admin to take in the case of such an event? Or is it just for statistics, then perhaps we need a metric so admins could plan ahead?